pkg/utils/markdown_utils.go

```diff
@@ -78,10 +80,12 @@ func PrintErrorMarkdownAndExit(title string, err error, suggestion string) {
 	os.Exit(1)
```

Comment:
```suggestion
        // TODO: Refactor so that we only call `os.Exit` in `main()` or `init()` functions. 
        // Exiting here makes it difficult to test.
        // revive:disable-next-line:deep-exit
	os.Exit(1)
```